healthcare/fhir_store_patch.go

```diff
@@ -36,11 +36,11 @@ func patchFHIRStore(w io.Writer, projectID, location, datasetID, fhirStoreID, to
 
 	name := fmt.Sprintf("projects/%s/locations/%s/datasets/%s/fhirStores/%s", projectID, location, datasetID, fhirStoreID)
 
-	if _, err := storesService.Patch(name, &healthcare.FhirStore{
-		NotificationConfig: &healthcare.NotificationConfig{
-			PubsubTopic: topicName, // format is "projects/*/locations/*/topics/*"
-		},
-	}).UpdateMask("notificationConfig").Do(); err != nil {
+	notificationConfig := &healthcare.FhirNotificationConfig{
+		PubsubTopic: topicName, // format is "projects/*/locations/*/topics/*"
+	}
+
+	if _, err := storesService.Patch(name, &healthcare.FhirStore{NotificationConfigs: []*healthcare.FhirNotificationConfig{notificationConfig}}).UpdateMask("notificationConfigs").Do(); err != nil {
```

Comment:
issue: initialize the `healthcare.FhirStore` object and call `UpdateMask()` outside of this block.

This is difficult to read as written.

Reply:
Done.